crates/biome_markdown_parser/src/token_source.rs

```diff
@@ -63,6 +62,30 @@ impl<'source> MarkdownTokenSource<'source> {
             }
         }
     }
+
+    /// Returns the number of whitespace characters before the current token until the first new line.
+    /// tab will be counted as 4 spaces
```

Comment:
Is there a link to the spec that says that? We need to justify certain decisions